README.md

```diff
@@ -107,7 +110,7 @@ include(FetchContent)
 FetchContent_Declare(
     Corrosion
     GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
-    GIT_TAG origin/master # Optionally specify a version tag or branch here
+    GIT_TAG origin/master # Optionally specify a commit hash, version tag or branch here
```

Comment:
should we actually use the latest stable tag here in that doc so people that copy paste from there get a stable setup?

Reply:
Makes sense to me. I set it to v0.1.0 for now.